src/forms/Form/Form.stories.tsx

```diff
@@ -142,6 +142,7 @@ export const Playground = (): ReactElement => {
 	const showSections = boolean("Show sections", false);
 	const prepopulateValues = object("Prepolulate values", {
 		"textField": "Text field prepopulated",
+		"numberField": 4596,

```

Comment:
I don't think we need to add a number example here, since the playground only has the most basic example of each field. Here we would have 2 variants of the same field.